### PR TITLE
add zh trans content/zh/docs/reference/setup-tools/kubeadm/generated/…

### PR DESCRIPTION
**Pull request by tanjunchen**

### CODE REVIEW

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_addon_coredns.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+<!--
+### Synopsis
+-->
+### 概要
+
+
+<!--
+Install the CoreDNS addon components via the API server. Please note that although the DNS server is deployed, it will not be scheduled until CNI is installed.
+-->
+通过 API 服务器安装 CoreDNS 附加组件。请注意，尽管已部署 DNS 服务器，但在安装 CNI 之前不会调度 DNS 服务器。
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_addon_coredns.md]:
```suggestion
通过 API 服务器安装 CoreDNS 附加组件。请注意，即使 DNS 服务器已部署，在安装 CNI 之前 DNS 服务器不会被调度执行。
```

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_addon_coredns.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+
+    <tr>
+      <td colspan="2">--feature-gates string</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">
+      <!--
+      A set of key=value pairs that describe feature gates for various features. Options are:<br/>IPv6DualStack=true|false (ALPHA - default=false)
+      -->
+      一组用来描述各种功能特性的键值（key=value）对。选项是：<br/>IPv6DualStack=true|false (ALPHA - default=false)
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_addon_coredns.md]:
```suggestion
      一组用来描述各种功能特性的键值（key=value）对。选项是：<br/>IPv6DualStack=true|false (ALPHA - 默认值=false)
```

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_bootstrap-token.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+<!--
+### Synopsis
+-->
+### 概要
+
+
+<!--
+Bootstrap tokens are used for establishing bidirectional trust between a node joining the cluster and a control-plane node.
+-->
+Bootstrap 令牌（token）用于在加入集群的节点和控制平面节点之间建立双向信任。
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_bootstrap-token.md]:
```suggestion
启动引导令牌（bootstrap token）用于在即将加入集群的节点和控制平面节点之间建立双向信任。
```

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_certs_apiserver.md [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+
+    <tr>
+      <td colspan="2">--apiserver-cert-extra-sans stringSlice</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">
+      <!--
+      Optional extra Subject Alternative Names (SANs) to use for the API Server serving certificate. Can be both IP addresses and DNS names.
+      -->
+      用于 API Server 服务证书的可选附加主题备用名称（SAN）。可以是 IP 地址和 DNS 名称。
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_certs_apiserver.md]:
```suggestion
      用于 API Server 服务证书的可选附加主体备用名称（SAN）。可以是 IP 地址和 DNS 名称。
```

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_certs_apiserver.md [hunk 5]
```diff
@@ -0,0 +1,240 @@
...
+      -->
+      --service-cidr string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;默认值："10.96.0.0/12"
+      </td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">
+      <!--
+      Use alternative range of IP address for service VIPs.
+      -->
+      指定服务 VIP 使用 IP 地址的其他范围。
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_certs_apiserver.md]:
```suggestion
      指定服务 VIP 可使用的其他 IP 地址段。
```

**Comment by tanjunchen** [comment]
@tengqm  已经修改

**Pull request merged**